datadog_checks_dev/datadog_checks/dev/plugin/pytest.py

```diff
@@ -138,7 +138,9 @@ def dd_environment_runner(request):
 @pytest.fixture
 def dd_agent_check(request, aggregator):
     if not e2e_testing():
-        pytest.skip('Not running E2E tests')
+        pytest.skip(
+            'Not running E2E tests. If this is a mistake please define DDEV_E2E_PYTHON_PATH and TOX_ENV_NAME env vars'
```

Comment:
Does `DDEV_E2E_PYTHON_PATH` need to be set manually ? Seems to be set here in e2e

https://github.com/DataDog/integrations-core/blob/c5501c0da4cd037404b292b89fca4221767e65e4/datadog_checks_dev/datadog_checks/dev/tooling/commands/test.py#L128

Reply:
You need to set it if you run the test from the IDE instead of with the ddev command

Reply:
@AlexandreYang Agreed with Julia, seems like `test_env_vars` is only populated if using `ddev`. For IDEs one needs to set it so that `e2e_testing` to return true:

https://github.com/DataDog/integrations-core/blob/5df7457ed4329f1dd0d41fa9781c0e3389174e15/datadog_checks_dev/datadog_checks/dev/_env.py#L10

https://github.com/DataDog/integrations-core/blob/5df7457ed4329f1dd0d41fa9781c0e3389174e15/datadog_checks_dev/datadog_checks/dev/_env.py#L14

https://github.com/DataDog/integrations-core/blob/5df7457ed4329f1dd0d41fa9781c0e3389174e15/datadog_checks_dev/datadog_checks/dev/_env.py#L26-L27

Reply:
Can you give me an example workflow? a few of us use an IDE shell & `ddev` works

Reply:
On a console I do the `ddev env start` then on the IDE I do right click + run test on the single test I want to run. It's really useful for debugging. When I do that I need to set up the env vars for that test run configuration:

![image](https://user-images.githubusercontent.com/611228/78031987-990e5a80-7364-11ea-81cf-74cf481f794a.png)

It works perfectly. It just I forget the vars I need to set so I need to dig through the code every time.

Reply:
We don't/can't support test invocations without `ddev`, it injects A LOT of required env vars and flags, not just those 2. If it's working it is only by luck and won't work in most other cases